tests/all_good_test.py

```diff
@@ -158,38 +166,38 @@ class TestAllGood(object):
         ("||x||", _Abs(_Abs(x))),
         ("|x||y|", _Abs(x) * _Abs(y)),
         ("||x||y||", _Abs(_Abs(x) * _Abs(y))),
-        ("\\lfloor x\\rfloor", floor(x)),
-        ("\\lceil y\\rceil", ceiling(y)),
-        ("\\pi^{|xy|}", pi**_Abs(x * y)),
+        ("\\lfloor x\\rfloor ", floor(x)),
+        ("\\lceil y\\rceil ", ceiling(y)),
+        ("\\pi ^{|xy|}", pi**_Abs(x * y)),
         ("\\frac{\\pi}{3}", _Mul(pi, _Pow(3, -1))),
-        ("\\sin{\\frac{\\pi}{2}}", sin(_Mul(pi, _Pow(2, -1)), evaluate=False)),
+        ("\\sin{\\frac{\\pi }{2}}", sin(_Mul(pi, _Pow(2, -1)), evaluate=False)),
         ("a+b\\imaginaryI ", a + I * b),
-        ("e^{\\imaginaryI \\pi}", exp(_Mul(I, pi), evaluate=False)),
-        ("e^\\pi", exp(pi, evaluate=False)),
-        ("\\int x dx", Integral(x, x)),
-        ("\\int x d\\theta", Integral(x, theta)),
+        ("e^{\\imaginaryI \\pi }", exp(_Mul(I, pi), evaluate=False)),
+        ("e^\\pi ", exp(pi, evaluate=False)),
+        ("\\int xdx", Integral(x, x)),
```

Comment:
@builtbywill and then in some cases here we are removing the spaces? I'm confused.

Reply:
TLDR, I tried to make the latex examples match what comes from mathlive